lib/logstash/devutils/rspec/shared_examples.rb

```diff
@@ -0,0 +1,23 @@
+require 'rspec/wait'
+
+RSpec.shared_examples "an interruptible input plugin" do
+  describe "#stop" do
+    let(:queue) { SizedQueue.new(20) }
+    subject do
+      s = described_class.new(config)
```

Comment:
@talevy @ph how do you pretend to make this more explicit to the users? would you prefer having a parameter ? by the end this should have documentation where this is noted, but using described class is not an issue per se. What do you think?